### ISSUE TITLE
Get people I'm following

### ISSUE THREAD

**Issue by eakron** [BoolNordicAB/sharepoint-utilities#27]
For a given username, retrieve all the users that user is following.

Assume we work with usernames (take a username, return a list of usernames) but if that is inconvenient we can use some ID or URL instead?

Eg:
```
sputils.user.getFollowing(username)
  .then(function (followers) {
    console.log(followers);
});
```

**Comment by roobie** [comment]
Good idea. Perhaps it can result in a list of objects with the username and a URL to more data á la SharePoint REST?
